0019-glossary/0019-glossary.md

```diff
@@ -236,6 +239,11 @@ See [IL-RFC 9](../0009-simple-payment-setup-protocol/0009-simple-payment-setup-p
 ## Source (transfer, ledger, amount)
 The transfer/ledger/amount directly adjacent to the sender.
 
+## Streaming Payment
```

Comment:
I like the term `Streaming`, but still feel like it still collides with meaning of a "chunked" payment (and can think of some examples that seem to fit both definitions).

Is a "Streaming Payment" actually a type of Chunked Payment, or are these meant to be distinct?

Reply:
It's possible that you would use the same protocols for both, but the idea was to separate when you're talking about paying for an ongoing or streaming service vs when you're trying to send a single larger amount and break it up for other reasons.